docs/geedocs/answer/6259403.html

```diff
@@ -10,6 +10,7 @@
 <!-- 6259403.html -->
 <div class="container">
 <div class="sidebar1">
+  <p class="sidebar-toc"><a href="../answer/7160000.html">Open GEE 5.2.0</a></p>
```

Comment:
It's pretty obvious that these docs were auto-generated.  I wonder what tool they used?